pkg/validation/cluster_test.go

```diff
@@ -89,10 +89,10 @@ func TestValidateContainerRuntime(t *testing.T) {
 			valid: true,
 		},
 		{
-			name: "use docker with 1.26",
+			name: "use docker with 1.28",
 			spec: &kubermaticv1.ClusterSpec{
 				ContainerRuntime: "docker",
-				Version:          *semver.NewSemverOrDie("1.26.0"),
+				Version:          *semver.NewSemverOrDie("1.28.0"),
```

Comment:
This change doesn't make sense IMHO. Docker is not supported in newer Kubernetes versions, so if anything this test should be removed or kept as is.